nifi-registry/nifi-registry-assembly/pom.xml

```diff
@@ -90,11 +90,11 @@
         </dependency>
         <dependency>
             <groupId>org.apache.nifi.registry</groupId>
-            <artifactId>nifi-registry-bootstrap</artifactId>
+            <artifactId>nifi-registry-runtime</artifactId>
```

Comment:
The order of dependencies matter for maven assembly plugin and the newly added `nifi-utils` dependency was put into the wrong directory (into lib/bootstrap instead of lib). Changing the order worked for me but feels a little fragile. I wonder if there's a better solution for this?